proposals/flexible-vectors/FlexibleVectors.md

```diff
@@ -0,0 +1,327 @@
+Flexible vectors overview
+=========================
+
+The goal of this proposal is to provide flexible vector instructions for
+WebAssembly as a way to bridge the gap between existing SIMD instruction sets
+available on various platforms. `simd128` proposal already identified
+operations that would commonly work on platforms that are important to
+WebAssembly, this proposal is attempting to extend the same operations to work
+with variable vector lengths.
+
+## Types
+
+Proposal introduces the following vector types:
+
+- `vec.v8` : 8-bit lanes
```

Comment:
May be the types can be referred to in this document as 'flexible vector' instead of 'vector' to differentiate them more against fixed width SIMD vectors. Also, something like fvec instead of vec.

Reply:
I filed #6 specifically for types, there is a bit more to it, in line with one of your other comments